### ISSUE TITLE
CHATHUB

### ISSUE THREAD

**Issue by bigguystevieb1** [bigguystevieb1/blookethack#6]
This Is An Issue Meant For @bigguystevieb1 & @ForkAndSpoonIt to talk

**Comment by bigguystevieb1** [comment]
@ForkAndSpoonIt are u there?

**Comment by bigguystevieb1** [comment]
@ForkAndSpoonIt If you can see this, I need to leave for a couple hours, I may still be able to respond to Issues, If anyone has any Issues can you respond to them, but don't delete them because I would like to look over them, I will be back in a couple hours. Thank You

**Comment by bigguystevieb1** [comment]
@ForkAndSpoonIt If you can see this, I need to leave for a couple hours, I may still be able to respond to Issues, If anyone has any Issues can you respond to them, but don't delete them because I would like to look over them, I will be back in a couple hours. Thank You

**Comment by ForkAndSpoonIt** [comment]
pl

**Comment by ForkAndSpoonIt** [comment]
ok

**Comment by ForkAndSpoonIt** [comment]
@bigguystevieb1  the guy who might want to join isnt all that good at code but eager to learn it

**Comment by bigguystevieb1** [comment]
hey @ForkAndSpoonIt

**Comment by ForkAndSpoonIt** [comment]
? im back @bigguystevieb1

**Comment by ForkAndSpoonIt** [comment]
@bigguystevieb1 We gotta make these from scratch

**Comment by ForkAndSpoonIt** [comment]
@bigguystevieb1  seems like a better idea. why would they get the hax from here instead of schoolcheats?

**Comment by bigguystevieb1** [comment]
@ForkAndSpoonIt

**Comment by ForkAndSpoonIt** [comment]
Yeah but theoristedu.org,artbanner.cf,qualitymath.us,brainstormbotus,degreelookup.org

**Comment by ForkAndSpoonIt** [comment]
@bigguystevieb1

**Comment by bigguystevieb1** [comment]
sorry man @ForkAndSpoonIt  ive went to the shore with my gf's family, im back

**Comment by bigguystevieb1** [comment]
i will try to be more active

**Comment by ForkAndSpoonIt** [comment]
ok

**Comment by bigguystevieb1** [comment]
i added another hack. are u working on hacks

**Comment by bigguystevieb1** [comment]
if not i dont see a reason for you to be a collaborator.

**Comment by ForkAndSpoonIt** [comment]
Im not that good at hacks. I want to join in the fight of making games sites @bigguystevieb1

**Comment by bigguystevieb1** [comment]
Ok, Thanks for your time goodbye @ForkAndSpoonIt
